src/data/matrix/basic.lean

```diff
@@ -1647,148 +1647,154 @@ lemma star_mul [fintype n] [non_unital_semiring α] [star_ring α] (M N : matrix
   star (M ⬝ N) = star N ⬝ star M := conj_transpose_mul _ _
 
 end star
+end matrix
```

Comment:
Why do you need to exit and reenter the matrix namespace?

Reply:
I suspect the answer is "to avoid notation problems", in which case you can use `\f<<on\f>>` as the name.